cmd/server-main.go

```diff
@@ -144,18 +144,20 @@ func initConfig() {
 
 // Generic Minio initialization to create/load config, prepare loggers, etc..
 func minioInit(ctx *cli.Context) {
-	// Is TLS configured?.
-	globalIsSSL = isSSL()
-
-	// Initialize minio server config.
-	initConfig()
-
 	// Enable all loggers by now so we can use errorIf() and fatalIf()
 	enableLoggers()
```

Comment:
There is a problem here.

Without initializing log, `errorIf()` and `fatalIf()` functions won't work.  There are functions which use those logger functions.

I will move back these statements like before and address the above problem later.